docs/patterns/single-new-eks-observability-accelerators/single-new-eks-native-observability.md

```diff
@@ -0,0 +1,137 @@
+# Single New EKS Cluster Native Observability Accelerator
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Single EKS Cluster Native Observability pattern using AWS native tools such as CloudWatch Logs and Container Insights:
```

Comment:
Could you talk a little bit about this architecture and why its AWS native? Also talk about Container Insights a bit?